clevis/Sanity/simple-bind-luks/runtest.sh

```diff
@@ -87,7 +87,11 @@ rlJournalStart
 
         rlRun "packageVersion=$(rpm -q ${PACKAGE} --qf '%{name}-%{version}-%{release}\n')"
         if rlTestVersion "${packageVersion}" '>=' 'clevis-15-8'; then
-            rlAssertGrep "Error communicating with the server http://localhost" $rlRun_LOG
+            if rlIsRHEL; then
+		rlAssertGrep "Error communicating with the server http://localhost" $rlRun_LOG
+            else
+		rlAssertGrep "Error communicating with server http://localhost" $rlRun_LOG
+            fi
```

Comment:
this might be a better solution:
```suggestion
            rlAssertGrep "Error communicating with .*server http://localhost" $rlRun_LOG -Eq
```

Reply:
Changed to proposed solution!